### PR TITLE
Add tenet blockchain

### PR DESCRIPTION
**Pull request by danil-lashin**

### CODE REVIEW

File: tenet/chain.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "tenet",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://tenet.org/",
+  "pretty_name": "Tenet",
+  "chain_id": "tenet_1559-1",
+  "bech32_prefix": "tenet",
```
Comment (JeremyParish69) [inline tenet/chain.json:9]: Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: tenet/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "description": "The native EVM, governance and staking token of the Tenet",
+      "denom_units": [
+        {
+          "denom": "atenet",
+          "exponent": 0
+        },
+      ],
+      "base": "atenet",
+      "name": "Tenet",
+      "display": "tenet",
```
Comment (JeremyParish69) [inline tenet/assetlist.json:15]: this display [denom] property is mean to point to a defined denom_unit which relays the number of decimal points to use. If the practically used unit TENET has zero exponent (is indivisible), then the display should be atenet, but I'm guessing from the 'a' that there should be another denom_unit called 'tenet' with exponent 18...

File: tenet/chain.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    "cosmos_sdk_version": "0.46",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "ibc_go_version": "6.1.0",
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/tenet-org/tenet-mainnet/master/config/genesis.json"
+    },
+    "versions": []
```
Comment (JeremyParish69) [inline tenet/chain.json:59]: Can you please also copy the version data you provided above into an object in this versions array?

File: tenet/chain.json (hunk 1 above)
Comment (danil-lashin) [reply tenet/chain.json:9]: Thanks, the pr has been created: https://github.com/satoshilabs/slips/pull/1560

**Pull request merged**